### PR TITLE
Use SOR to query orbitReset timestamp for Run2

### PR DESCRIPTION
**Pull request by ekryshen**

### CODE REVIEW

**Comment by jgrosseo** [comment]
Thanks!
This does not look right: 
image

**Comment by jgrosseo** [comment]
And this? 
image

File: Common/TableProducer/timestamp.cxx [hunk 1]
```diff
@@ -85,29 +85,35 @@ struct TimestampTask {
...
 
-    timestampTable((orbitResetTimestamp + int64_t(bc.globalBC() * o2::constants::lhc::LHCBunchSpacingNS * 1e-3)) / 1000); // us -> ms
+  if (verbose.value) {
+    LOGF(info, "Orbit-reset timestamp for run number %i found: %llu us", runNumber, orbitResetTimestamp);
   }
-};
+
+  timestampTable((orbitResetTimestamp + int64_t(bc.globalBC() * o2::constants::lhc::LHCBunchSpacingNS * 1e-3)) / 1000); // us -> ms
+}
+}
```
Comment (jgrosseo) [inline Common/TableProducer/timestamp.cxx:115]:
```suggestion
};
```

File: Common/TableProducer/timestamp.cxx [hunk 2]
```diff
@@ -85,29 +85,35 @@ struct TimestampTask {
...
+;
```
Comment (jgrosseo) [inline Common/TableProducer/timestamp.cxx:116]:
```suggestion
};
```

Comment (ekryshen) [reply Common/TableProducer/timestamp.cxx]: Yes, I am not sure how I managed to make these bugs. It was so simple...

**Review by jgrosseo — APPROVED** [review]

**Comment by ekryshen** [comment]
Too many brackets... Need another fix

**Comment by TimoWilken** [comment]
Hi @ekryshen, it appears you're introducing an additional line containing only a `}` in this PR, with all lines after it outdented by one level.

This additional line is more easily visible if you hide whitespace changes (in GitHub's web UI, go to Files changed -> cog icon -> Hide whitespace -> Apply and reload; alternatively, use `git diff --ignore-space-change` on the command line).

Is this intentional? I can't find a balancing `{` for the additional `}` in the changed lines.

**Comment by ekryshen** [comment]
Hi @TimoWilken, yes, fixed, thanks for checking!

**Comment by TimoWilken** [comment]
CI is complete; merged.

**Pull request merged**